common/src/main/java/whocraft/tardis_refined/common/items/ZeitonIngotItem.java

```diff
@@ -0,0 +1,37 @@
+package whocraft.tardis_refined.common.items;
+
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.context.UseOnContext;
+import net.minecraft.world.level.block.Blocks;
+import whocraft.tardis_refined.registry.BlockRegistry;
+
+public class ZeitonIngotItem extends Item {
+    public ZeitonIngotItem(Properties properties) {
+        super(properties);
+    }
+
+    @Override
+    public InteractionResult useOn(UseOnContext useOnContext) {
+
+
+
+        if (useOnContext.getLevel().getBlockState(useOnContext.getClickedPos()).getBlock() == Blocks.IRON_BLOCK) {
```

Comment:
getLevel is used numerous times, make it a variable

Reply:
Made both a variable